meshers/blocky/voxel.cpp

```diff
@@ -190,6 +190,8 @@ void Voxel::set_collision_mask(uint32_t mask) {
 static void bake_cube_geometry(Voxel &config, Voxel::BakedData &baked_data, int p_atlas_size) {
 	const float sy = 1.0;
 
+	std::vector<Vector3> deltaPos;
```

Comment:
This function is specialized for cubes. You don't need a dynamic vector. You could use a `FixedArray<Vector3, Cube::SIDE_COUNT * 3>` instead.
But even then, I'm wonder why you need to hold data for every side at once. Each face is an independent quad, so I'm sure you could use an array of 3 elements, or a lookup table of precalculated tangents.